cronofy.php

```diff
@@ -445,6 +445,33 @@ public function change_participation_status($params)
         return $this->http_post("/" . self::API_VERSION . "/calendars/" . $params["calendar_id"] . "/events/" . $params["event_uid"] . "/participation_status", $postfields);
     }
 
+    public function availability($params)
+    {
+        /*
+          participants : Array of member arrays and the required number of people to be available from each array. REQUIRED
```

Comment:
Could just lift this from [the docs](https://www.cronofy.com/developers/api/alpha/#availability):

> An array of the groups of participants whose availability should be taken into account